examples/site/howto_local_development/README.md

```diff
@@ -34,25 +34,86 @@ would use:
 
 ```shell
 cd $HOME
-git clone https://github.com/microsoft/vcpkg
-(cd vcpkg && ./bootstrap.sh)
+git clone -q https://github.com/microsoft/vcpkg
+# Expected output: none
+./vcpkg/bootstrap-vcpkg.sh --disableMetrics --useSystemBinaries
 ```
 
+You should see output like this:
+
+```console
+Downloading vcpkg tool sources
+...
+..
+Building vcpkg-tool...
+...
+..
+[88/88] Linking CXX executable vcpkg
+```
+
+This will create a `vcpkg` executable in the `$HOME/vcpkg` directory.
+
 ## Compiling a function
 
-Once vcpkg is compiled you can build the example using:
+Once vcpkg is compiled you can build the example. If you have not cloned
+the `functions-framework-cpp` repository yet, you need to do so now:
 
-> :warning: the first time you build the framework it might take several
-> minutes, maybe as much as 1/2 hour, depending on your workstation
-> performance. Future builds, even in different directories, should be
-> faster as `vcpkg` caches binary artifacts in `$HOME/.cache/vcpkg`.
+```shell
+cd $HOME
+git clone -q https://github.com/GoogleCloudPlatform/functions-framework-cpp.git
+# Expected output: none
+```
+
+Then go to the directory hosting this example and use CMake to build
+the example:
+
+```shell
+cd functions-framework-cpp/examples/site/howto_local_development
+```
+
+Run the CMake configure step. If needed, this will download and build any
+dependencies for your function:
 
 ```shell
-cd $HOME/functions-framework-cpp/examples/site/howto_local_development
 cmake -H. -B.build -DCMAKE_TOOLCHAIN_FILE=$HOME/vcpkg/scripts/buildsystems/vcpkg.cmake
-cmake --build .build -- -j $(nproc)
 ```
 
+You should see output like this:
+
+> :warning: the first time you use or build the framework it might take several
+> minutes, maybe as much as 1/2 hour, depending on your workstation
+> performance. Future builds, even in different directories, should be faster
+> as `vcpkg` caches binary artifacts in `$HOME/.cache/vcpkg`.
+
+```console
+-- Running vcpkg install
+...
+-- Running vcpkg install - done
+-- Configuring done
+-- Generating done
```

Comment:
Note, I see this:

```
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/timmerman/functions-framework-cpp/examples/site/howto_local_development/.build
```

Just noting that, I don't think we require changes.

Reply:
Fixed anyway.